packages/react-app/src/App.jsx

```diff
@@ -93,18 +129,56 @@ function App({ mainnet, localProvider, appChainId }) {
     }
     const provider = window.ethereum
     // @ts-ignore
-    setInjectedProvider(new ethers.providers.Web3Provider(window.ethereum))
+    // setInjectedProvider(new ethers.providers.Web3Provider(window.ethereum))
+    appDispatch({ actionType: APPSTATEACTION.OPTIMISMCHAINID })
```

Comment:
by default you dispatch `OPTIMISMID` ?

Reply:
Default is goerli actually. But I see what you are saying here. I am refactoring the code so one of these methods is left.